msresist/expectation_maximization.py

```diff
@@ -37,6 +37,11 @@ def EM_clustering(data, info, ncl, seqDist=None, gmmIn=None):
     # Add a dummy variable for the sequence information
     d = np.hstack((d, idxx.T))
 
+    # Setup weights for distributions
+    seqWarr = np.ones(d.shape[1])
+    seqWarr[-1] = seqWeight
+    seqWarr /= np.sum(seqWarr)
```

Comment:
This means that the sequence contribution doesn't explode. When the sequence contribution becomes large (e.g., `seqWeight = 100.0`, the Gaussians are scaled down appropriately.

Reply:
this provides a weight per feature (e.g. 10 AXL mutant cell lines), right? if so why does the last cell line get `seqWeight`?

Reply:
Because the sequence distribution is tacked onto the end as an additional "feature".

Reply:
Oh true got it.